src/components/vmd-appointment-card.component.ts

```diff
@@ -30,12 +30,28 @@ export class VmdAppointmentCardComponent extends LitElement {
         super();
     }
 
+    private clickEventHandler(e: Event, navigateToUrlIfPossible: boolean) {
+        if (navigateToUrlIfPossible) {
+            Router.navigateToUrlIfPossible(this.lieu.url)
```

Comment:
question bête, pourquoi on utilise pas des balises `<a>` ?

Reply:
Étant donné qu'on utilise un seul composant entre les lieux disponibles et lieux non disponibles, `this.lieu.url` peut être `undefined`

Dans le cas où l'url n'est pas définie, on ne souhaite pas trigger de navigation si l'utilisateur clique sur la card (c'est ce qu'apporte `Router.navigateToUrlIfPossible()` comme plu-value)

Ça prépare également le terrain dans le cas où on souhaiterait faire un évènement GA sur click sur le bouton de prise de rdv

Reply:
À la lecture du code qui était déjà en place, l'objectif est que le clic sur n'importe quel point de la carte de centre redirige l'utilisateur vers l'url. Une balise `<a>` est de type inline alors que les `<div>` sont des blocs et au niveau de la carte, on a des `<div>` uniquement. Je suppose que c'est la raison pour laquelle un event sur le div a été fait plutôt qu'un `<a>`. Mais je laisse @fcamblor confirmer.